src/python/ovms_py_tensor.hpp

```diff
@@ -64,7 +65,7 @@ const std::unordered_map<std::string, std::string> bufferFormatToDatatype{
     // {X, "BF16"} to be considered, for now it shall be treated as a custom datatype
 };
 
-// TO DO: Note that for numpy for example np.int64 gets translates to "l" not "q" on 64 bit linux systems.
+// TO DO: Note that for numpy for example np.int64 translates to "l" not "q" on 64 bit linux systems.
```

Comment:
Use TODO instead of TO DO to make our release greps find what we want

Reply:
Will do